### ISSUE TITLE
cx-doc - project

### ISSUE THREAD

**Issue by ahebert-lt** [ahebert-lt/cx#70]
once the interpreter is complete a language documentation project will be sarted

**Comment by ahebert** [comment]
Args:
-d <output filename>
--documentation <output filename>

Implementation:
Documentation should be parsed out of comment blocks similar to doxygon comments.
Documentation string should be associated with the following tokens symbol table node.

Symbol table nodes now need a std::wstring member to contain text for documentation.

/*! 
DocumentationHeader - BriefDescription
        Body text. Followed by more meta-data. 
@param: Parameter1
@param: Parameter2
@return: ReturnValue 
*/

**Comment by ahebert-lt** [comment]
Closing legacy issue - CX Language has evolved into a cognitive programming platform with native AI integration. This legacy feature is no longer relevant to the current architecture.
